src/BenchmarkDotNet.Diagnostics.Windows/Sessions.cs

```diff
@@ -71,6 +71,8 @@ internal override Session EnableProviders()
             if (Details.Config.GetHardwareCounters().Any())
                 keywords |= KernelTraceEventParser.Keywords.PMCProfile; // Precise Machine Counters
 
+            TraceEventSession.StackCompression = true;
```

Comment:
it would be great if you could add some comment on why we need that

Reply:
To be honest, we do not need it, but PerfView uses it and as you can read here: https://gist.github.com/Zhentar/7f3f60fb51e89fe95abca7b9724964ba:

>"Stack Caching" (or Stack Compression as PerfView calls it) is a feature of ETW designed to reduce trace buffer & etl file sizes by de-duplicating stack traces. 

I can revert it if you want.